### ISSUE TITLE
Model Method Names

### ISSUE THREAD

**Issue by stanistan** [SkyPHP/skyphp#76]
The Model class has these hooks for save:
- `before_insert()`
- `after_insert()`
- `before_update()`
- `after_update()`

---

There are two methods `after_save()` and `after_fail()` which share a similar name, but are used for vastly different things.

Each returns the `response_array` which has this structure:

``` php
<?php

array(
    'status' => $status,
    'errors' => $errors,
    'data' => $data
);

```

I want to rename these methods... to `successResponse` and `errorResponse` respectively.

Thoughts?

---

**Note:** They are only being used a handful of times outside of `class.Model.php` and would be very quick to change.

**Comment by will123195** [comment]
do it

**Comment by stanistan** [comment]
Closed by #84
